sdcm/cluster.py

```diff
@@ -2048,7 +2048,8 @@ def offline_install_scylla(self, unified_package, nonroot):
         elif self.distro.is_sles:
             raise Exception("Offline install on SLES isn't supported")
         elif self.distro.is_debian10 or self.distro.is_debian11:
-            self.remoter.run(f'sudo apt-get install -y openjdk-11-jre-headless {additional_pkgs}')
+            self.remoter.sudo('apt-get install -y openjdk-11-jre')
+            self.remoter.sudo(f'apt-get install -y openjdk-11-jre-headless {additional_pkgs}')
```

Comment:
Do we need both? ` openjdk-11-jre` and `openjdk-11-jre-headless`

Reply:
technically speaking, i don't think we need both, but i have added kind of a workaround to the `ca-certificate` issue, that Debian is supposed to be fixing, and once it will be fixed, then we can remove this new line, and leave the `openjdk-11-jre-headless` only

Reply:
i can add one, but i don't have any specific issue to link to

Reply:
@mykaul https://github.com/scylladb/scylla-cluster-tests/pull/6748